Client/src/components/pages/Dashboard/AdminDashboard/AdminCategories/index.jsx

```diff
@@ -68,7 +64,15 @@ const AdminCategories = () => {
             <div className="flex justify-end mb-4">
                 <Link to="/admin-createCategories" className="inline-block px-4 py-2 bg-green-500 text-white rounded hover:bg-green-600 mr-20">+ ADD CATEGORY</Link>
             </div>
-            <Table data={categories} handleUpdate={handleUpdate} handleProductDelete={handleProductDelete} type="category" />
+            {/* <Table data={categories} handleUpdate={handleUpdate} handleProductDelete={handleProductDelete} type="category" /> */}
+            <Table data={categories}
+                handleUpdate={handleUpdate}
+                handleDelete={handleDelete}
+                headers={[
+                    { key: 'id', label: ' ID' },
```

Comment:
use variable to store this array and pass it as a prop value in next iteration refactor it

Reply:
Okay, Noted, Applying the changes and Re-pushing the code.